scripts/format.sh

```diff
@@ -7,32 +7,24 @@ set -o errexit -o pipefail -o nounset
 #  - 1 there are files to be formatted
 #  - 0 everything looks fine
 
-
-readonly CLANG_FORMAT_VERSION=11.0.0
+# see https://pypi.org/project/clang-format/#history
+readonly CLANG_FORMAT_VERSION=11.0.1
 
 if [[ $(uname -i) == 'aarch64' ]]; then
   echo 'Formatting is disabled on arm for the time being'
   exit
 fi
 source scripts/bash_utils.sh
-setup_mason
-
-./mason/mason install clang-format $CLANG_FORMAT_VERSION
-./mason/mason link clang-format $CLANG_FORMAT_VERSION
```

Comment:
Is it possible to completely retire `mason`? It is only used in `scripts/format.sh` and `scripts/clang-tidy-only-diff.sh` for `clang-format` and `clang-tidy`. The `scripts/bash_utils.sh` would also get simplified by removing `setup_mason` function.